public/app/app.js

```diff
@@ -53,6 +53,11 @@
         }
       }).then(function (response) {
         $scope.title = response.data;
+	// Set default options for season and episode
+	if ($scope.title.seasons && $scope.title.seasons.length > 0) {
```

Comment:
does lodash have something like _.isNotEmpty for arrays? If yes, we could use it then you can merge. If not, go ahead and merge.